airbyte-integrations/connectors/source-goldcast/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "source-goldcast"
+description = "Source implementation for goldcast."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/goldcast"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+packages = [ { include = "source_goldcast" }, {include = "main.py" } ]
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
```

Comment:
Can you please bump this to "< 3" and run `poetry lock`? I'd love to start this on the fresh CDK. I will merge if test pass without it, but please follow up a PR to upgrade.

Reply:
Corrected ! @natikgadzhi